lib/modules/commentPreview.js

```diff
@@ -1,7 +1,7 @@
 addModule('commentPreview', {
 	moduleID: 'commentPreview',
 	moduleName: 'Live Comment Preview',
```

Comment:
Possibly worth just going with "Live Preview"? 
My suspicion would be that a lot of RES user's probably don't know what Snudown is.